test/Libraries/PackageManagerTests/PackageManagerClientTests.cs

```diff
@@ -397,6 +397,33 @@ public void Deprecate_ReturnsFailureObjectWhenRequestThrowsException()
             Assert.IsFalse(res.Success);
         }
 
+        [Test]
+        public void DoesCurrentUserOwnPackage_ReturnsFalseWhenCurrentUserIsNotTheOwner()
+        {
+            var id = "1";
+            var username = "abcd";
+
+            var mp = new ResponseWithContentBody<PackageHeader>
+            {
+                content = new PackageHeader()
+                {
+                    _id = id,
+                    maintainers= new List<User>()
+                },
+                success = true
+            };
+
+            var c = new Mock<IGregClient>();
+            c.Setup(x =>
+                x.ExecuteAndDeserializeWithContent<PackageHeader>(It.IsAny<HeaderDownload>()))
+                .Returns(mp);
+
+            var pc = new PackageManagerClient(c.Object, MockMaker.Empty<IPackageUploadBuilder>(), "");
+            var res = pc.DoesCurrentUserOwnPackage(new Package("1","1","2.0.4","1"), username);
+
+            Assert.IsFalse(res);
```

Comment:
not to be pedantic - but can we quickly add a positive test? (testing if the result is true when the maintainers list contains a matching username?